Description: Change a path in the Postgres output bot's README
 The Python script referenced here will be installed in PATH.
Author: Gernot Schulz <gernot@intevation.de>
Last-Update: 2016-07-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/intelmq/bots/outputs/postgresql/README.md
+++ b/intelmq/bots/outputs/postgresql/README.md
@@ -1,7 +1,7 @@
 
 # How to install:
 
-Use `intelmq/bin/intelmq_psql_initdb.py` to create initial sql-statements from Harmonization.conf
+Use `intelmq_psql_initdb` to create initial sql-statements from Harmonization.conf
 the script will create the required table layout and save it as /tmp/initdb.sql
 
 Now create a database-user and a database:
